recipes-extended/jailhouse/jailhouse_0.11.bb

```diff
@@ -0,0 +1,115 @@
+SUMMARY = "Jailhouse, i.MX fork"
+HOMEPAGE = "https://github.com/siemens/jailhouse"
+SECTION = "jailhouse"
+LICENSE = "GPL-2.0 & BSD-2-Clause"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=9fa7f895f96bde2d47fd5b7d95b6ba4d \
+                 file://tools/root-cell-config.c.tmpl;beginline=6;endline=33;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/jailhouse/hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/jailhouse/cell-config.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/arch/arm/asm/jailhouse_hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/arch/arm64/asm/jailhouse_hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://include/arch/x86/asm/jailhouse_hypercall.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+                 file://driver/jailhouse.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
+"
+
+SRCBRANCH = "lf-5.4.y"
+IMX_JAILHOUSE_SRC ?= "git://source.codeaurora.org/external/imx/imx-jailhouse.git;protocol=ssh"
```

Comment:
Just for clarity: why did you extracted `SRCBRANCH` and `IMX_JAILHOUSE_SRC` and didn't put them directly in `SRC_URI`?

Do you intend to override them later?
In that case, the `SRCBRANCH` should be weakly-defined (like `IMX_JAILHOUSE_SRC`).

Reply:
Indeed, for override. It is easier to maintain recipes when we develop software internally, then publish it externally to different mirrors.
So it's not weak for branch, will keep it as is.
Anyway, it doesn't look that bad for me, it's pretty clear.

Reply:
> It is easier to maintain recipes when we develop software internally, then publish it externally to different mirrors.

Got it. So you override this in your development environment to internal GIT and use a weak definition in the recipe.

> So it's not weak for branch, will keep it as is.

But I guess in this case it does not hurt to convert it to a weak definition as well, just in case if you would like to switch the branch internally as well. This would also bring consistency here, IMHO.

> Anyway, it doesn't look that bad for me, it's pretty clear.

I didn't say it's bad, was just a bit unclear why URI is weak and BRANCH is not.

Reply:
OK, not a problem

Reply:
actually, we can go without weak definitions